lib/Conversion/TorchToLinalg/IndirectDataMovement.cpp

```diff
@@ -222,23 +222,9 @@ class ConvertAtenEmbeddingBagPaddingIdxOp
     Value weight = adaptor.getWeight();
     Value indices = adaptor.getIndices();
     Value offsets = adaptor.getOffsets();
-    Value scaleGradByFreq = op.getScaleGradByFreq();
     Value mode = op.getMode();
-    Value sparse = op.getSparse();
```

Comment:
Hi @vivekkhandelwal1 ,
These are training-only specific arguments, and the existing implementation doesn't have the lowering for training-specific logic. We can introduce these arguments back if we ever decide to add lowering logic for training. However, I don't know if there is a use case for this though. Please note that I have only removed these arguments from the lowering as they were unused after my change. However, the tablegen file corresponding to this operator still has these arguments.

Reply:
Hi @Hanumanth04, since they are training-specific args how come they were set for instances where only eval/inference is done? I'm not able to understand if they are only training specific args then any code doing inference should not be setting them and if that's not true then the issue is where that PyTorch code is written at higher level not at the lowering level.